frontend/src/employee-frontend/components/employee/EmployeePinCodePage.tsx

```diff
@@ -21,6 +21,24 @@ import { isPinCodeLocked, updatePinCode } from '../../api/employees'
 import { useTranslation } from '../../state/i18n'
 import { useWarnOnUnsavedChanges } from '../../utils/useWarnOnUnsavedChanges'
 
+const badPins = [
```

Comment:
Setting a bad PIN disables the send button but does not explain why it is disabled

Reply:
Fixed!